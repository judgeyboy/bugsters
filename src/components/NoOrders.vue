<template>
  <div class="empty">
    <div class="empty-icon">
      <i class="icon icon-3x icon-link"></i>
    </div>
    <p class="empty-title h5">You don't have any orders!</p>
    <p class="empty-subtitle">
      Click the button to submit an order with us
    </p>
    <div class="empty-action">
      <router-link class="btn btn-primary" to="/">Submit an order</router-link>
    </div>
  </div>
</template>

<script>
export default {
  name: 'NoOrders'
}
</script>

