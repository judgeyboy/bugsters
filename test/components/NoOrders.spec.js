import Vue from 'vue'
import VueRouter from 'vue-router'
import {shallow} from 'vue-test-utils'

import NoOrders from '@/components/NoOrders'

Vue.use(VueRouter)

describe('SideBar component', () => {
  it('should be a vue instance', () => {
    const wrapper = shallow(NoOrders)
    expect(wrapper.isVueInstance()).toBe(true)
  })

  it('should match snapshot', () => {
    const wrapper = shallow(NoOrders)
    const $html = wrapper.vm.$el.outerHTML
    expect($html).toMatchSnapshot()
  })
})
