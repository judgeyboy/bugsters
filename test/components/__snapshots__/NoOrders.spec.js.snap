// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideBar component should match snapshot 1`] = `

<div class="empty">
  <div class="empty-icon">
    <i class="icon icon-3x icon-link">
    </i>
  </div>
  <p class="empty-title h5">
    You don't have any orders!
  </p>
  <p class="empty-subtitle">
    Click the button to submit an order with us
  </p>
  <div class="empty-action">
  </div>
</div>

`;
